source/aggregation-tutorials/filtered-subset.txt

```diff
@@ -0,0 +1,194 @@
+.. _node-aggregation-filtered-subset:
+
+===============
+Filtered Subset
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, sort, limit, aggregation
```

Comment:
Q: are the meta tags supposed to be directly under the heading? this is how I have seen it in other docs, but again not sure if it matters

Reply:
I'm not sure if there's a specific place other than at the top of the file, I just went with what I saw in other pages that we've added taxonomy tagging to.